.github/workflows/tooling.yml

```diff
@@ -68,4 +68,6 @@ jobs:
         export ASAN_OPTIONS=detect_stack_use_after_return=1:detect_leaks=1:check_initialization_order=true:strict_init_order=true:detect_stack_use_after_scope=1:fast_unwind_on_malloc=0
         export LSAN_OPTIONS=suppressions="$SOURCEPATH/.github/ci/sanitizer/clang/Leak.supp"
         export LD_PRELOAD=/usr/lib/clang/10/lib/linux/libclang_rt.asan-x86_64.so
-        ctest --test-dir build --output-on-failure
+        ctest --test-dir build -E 3b --output-on-failure
+        export OPENPMD_HDF5_CHUNKS="auto"
+        ctest --test-dir build -R 3b --output-on-failure
```

Comment:
Why does this test require changing things in the tooling? Shouldn't our Cmake cover this?